recipe/emailpassword/emaildelivery/smtpService/main.go

```diff
@@ -0,0 +1,40 @@
+package smtpService
+
+import (
+	"errors"
+
+	"github.com/supertokens/supertokens-golang/ingredients/emaildelivery"
+	"github.com/supertokens/supertokens-golang/supertokens"
+
+	evsmtpService "github.com/supertokens/supertokens-golang/recipe/emailverification/emaildelivery/smtpService"
+)
+
+func MakeSmtpService(config emaildelivery.SMTPTypeInput) emaildelivery.EmailDeliveryInterface {
+	serviceImpl := makeServiceImplementation(config.SMTPSettings)
+
+	if config.Override != nil {
+		serviceImpl = config.Override(serviceImpl)
+	}
+
+	emailVerificationServiceImpl := evsmtpService.MakeSmtpService(config)
```

Comment:
You need to pass an override here which will be provided an email verification service implementation (you need to create this in this folder).

That new service implementation should take `serviceImpl` as the input.